osu.Game/Screens/Edit/Timing/DifficultySection.cs

```diff
@@ -28,7 +28,16 @@ protected override void OnControlPointChanged(ValueChangedEvent<DifficultyContro
         {
             if (point.NewValue != null)
             {
-                multiplierSlider.Current = point.NewValue.SpeedMultiplierBindable;
+                var selectedPointBindable = point.NewValue.SpeedMultiplierBindable;
+
+                // there may be legacy control points, which contain infinite precision for compatibility reasons (see LegacyDifficultyControlPoint).
+                // generally that level of precision could only be set by externally editing the .osu file, so at the point
+                // a user is looking to update this within the editor it should be safe to obliterate this additional precision.
+                double expectedPrecision = new DifficultyControlPoint().SpeedMultiplierBindable.Precision;
```

Comment:
Not sure about eating an entire allocation for this each time instead of extracting the precision out to a const.

Reply:
it requires a user interaction (click selection) so not too important￼. but a const may look nicer.